src/ExpressiveInstaller/config.php

```diff
@@ -39,7 +40,7 @@
     'questions' => [
         'container' => [
             'question'               => 'Which container do you want to use for dependency injection?',
-            'default'                => 3,
+            'default'                => 4,
```

Comment:
Instead of changing the option, move `northwoods/container` to the last key. This also avoids breaking existing automation by consumers of this package.